floorplan-cli/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'application'
+    id 'java'
+    id 'org.jetbrains.kotlin.jvm'
+}
+
+group 'zynger'
+version '0.1-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+application {
+    mainClassName = javaMainClass
```

Comment:
Since this is the only gradle script that receives the `application` plugin, I think it would make more sense to inline the value of `javaMainClass` here and remove it from the root `build.gradle` file

Reply:
Yeha. This makes sense! 👍 Done in a35bf9b